src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/memory.cpp

```diff
@@ -6,10 +6,57 @@
 
 #include <vector>
 
+#include "ngraph_functions/builders.hpp"
+#include "openvino/op/util/variable.hpp"
+
 using namespace LayerTestsDefinitions;
+using namespace ngraph;
+using namespace opset7;
```

Comment:
Is it really needed? Or maybe you need opset10?

Reply:
if not explicitly given, we'd have to put it in front of each layer. opset11 has just been used

Reply:
My question was about opset7 - do you need opset7 specifically or did you just meant the newest opset?

Reply:
I didn't need opset7 specifically. Tests are also working with opset11 (newest) so I've used opset11.